docs/en/stack/security/authorization/overview.asciidoc

```diff
@@ -2,9 +2,9 @@
 [[authorization]]
 == User authorization
 
-{security} introduces the concept of _authorization_ to the {stack}.
-Authorization is the process of determining whether the user behind an incoming
-request is allowed to execute the request. 
+The {stack-security-features} introduce the concept of _authorization_ to the
+{stack}. Authorization is the process of determining whether the user behind an
```

Comment:
The repeated use of "stack" feels a little off. We may need to phrase this quite differently if we to avoid describing {security} as being something extra that's added to the stack.
Maybe:
```
The {stack-security-features} include support for the concept of _authorization_.
```
?

Reply:
Thanks @tvernum , I change it to: "The {stack-security-features} add _authorization_, which is the process of determining whether the user..."